builtin/audit/file/backend_test.go

```diff
@@ -0,0 +1,75 @@
+package file
+
+import (
+	"os"
+	"strconv"
+	"testing"
+
+	"github.com/hashicorp/vault/audit"
+	"github.com/hashicorp/vault/helper/salt"
+)
+
+func TestAuditFile_fileModeNew(t *testing.T) {
+	salter, _ := salt.NewSalt(nil, nil)
+
+	modeStr := "0777"
+	mode, err := strconv.ParseUint(modeStr, 8, 32)
+
+	file := "auditTest.txt"
```

Comment:
Sure, you could always close/delete the file and keep the unique name. Or you can use a tempdir and wipe the dir after the fact, then choose whatever files you want inside of it.